src/dev/storybook/aliases.ts

```diff
@@ -31,6 +30,7 @@ export const storybookAliases = {
   expression_shape: 'src/plugins/expression_shape/.storybook',
   expression_tagcloud: 'src/plugins/chart_expressions/expression_tagcloud/.storybook',
   fleet: 'x-pack/plugins/fleet/.storybook',
+  home: 'src/plugins/home/.storybook',
```

Comment:
Is this line supposed to add it to the published list? https://ci-artifacts.kibana.dev/storybooks/pr-135472/50f9115f7dbaaa0962a3ffcf3b768ccc2218c7ce/index.html

I was going to do a quick check on the render, but I'm not seeing a `home` link in the list.

Reply:
Nice catch... there's another change to make to add it to CI... will fix.